kong-2.3.0-0.rockspec

```diff
@@ -36,7 +36,7 @@ dependencies = {
   "lua-resty-cookie == 0.1.0",
   "lua-resty-mlcache == 2.5.0",
   "lua-messagepack == 0.5.2",
-  "lua-resty-openssl == 0.6.9",
+  "lua-resty-openssl == 0.6.10",
```

Comment:
```suggestion
  "lua-resty-openssl == 0.6.10-2",
```

context as to why: https://github.com/fffonion/lua-resty-openssl/commit/ea53cf407c9517ec9f5d25ea85de91602bb708b2

Reply:
@murillopaula, version should be enough. At least with my `luarocks` it installs the `-2`.

Reply:
Though the tests are failing as we don't have that module installed, so perhaps I need to bump build tools too.

Reply:
@bungle is right, please do not include rockspec revision numbers. They only mean that the rockspec changed, but not the source code. LuaRocks will automatically pick the latest one.

Reply:
Maybe the old version is cached?

Reply:
The test assets are available for introspection if one suspect CI built thing incorrectly

Steal the image tag from [here](https://github.com/Kong/kong/pull/6740/checks?check_run_id=1722826175#step:6:19)
![image](https://user-images.githubusercontent.com/697188/104952238-18bbf300-5992-11eb-9a80-9302efd3ee4b.png)

and run it
```
docker run -it --rm mashape/kong-build-tools:test-4dd8e8b-0-538950b-03 /bin/bash
luarocks list --porcelain | grep lua-resty-openssl
lua-resty-openssl       0.6.10-2        installed       /usr/local/lib/luarocks/rocks-5.1
```

BIG DISCLAIMER: how this will behave on systems where folks do an in place upgrade I cannot say